### PR TITLE
WCPayments task is not visible after installing the plugin

### PR DESCRIPTION
**Pull request by octaedro**
This PR aims to fix the problem mentioned in [this comment](https://github.com/woocommerce/woocommerce/issues/32330#issuecomment-1082928199). After installing WCPayments during the profiler, the WCPayments task should be visible instead of the task `Set up payments`.

It also adds the code to disable the installed plugins in the `Business Details` step in the OBW.

### Screenshots

screenshot-medieval-mantis jurassic ninja-2022 03 30-16_45_06

screenshot-four local-2022 03 30-19_18_43

### Detailed test instructions:

- Create a brand new store (e.g.: using JN).
- Create a zip (using `npm run test:zip`), install it, and activate it.
- In the OBW, during the Business Details, install WCPayments.
- After finishing the OBW the task `Get paid with WooCommerce Payments` should appear instead of the `Set up payments` task.
- Now go back to the step `Business Details > Free features`.
- Verify that the checkbox for `WooCommerce Payments` is disabled.
- Finish the OBW.
- Verify the task `Get paid with WooCommerce Payments` is still there.

### UPDATE
Since the checkbox won't be disabled in this PR, discard the testing step:
> Verify that the checkbox for `WooCommerce Payments` is disabled.

### CODE REVIEW

**Comment by octaedro** [comment]
@pmcpinto and @jarekmorawski, what do you think about disabling the WCPayments checkbox after installing the extension?

File: client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js [hunk 1]
```diff
@@ -339,6 +369,7 @@ export const SelectiveExtensionsBundle = ( {
 									)
 								);
 							} }
+							disabled={ installedExtensions.length !== 0 }
```
Comment (louwie17) [inline client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js]: It's a little confusing to me to have the `main` checkbox disabled when only one of the nested checkboxes is disabled. Logically this does make sense, but this is especially confusing when the dropdown is collapsed and the user might feel forced to install all the free extensions (although they only have one already installed):
Screen Shot 2022-03-31 at 9 57 51 AM

**Review by louwie17 — COMMENTED** [review]
Tested well, thanks for fixing this @octaedro 
I do think we might be better off removing the disabled checkbox logic for now and create an issue for that in core and fix it for the next release.
Keeping this fix more specific in relation to the WooCommerce Payments task not showing.

File: client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js (hunk 1 above)
Comment (octaedro) [reply client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js]: Sounds good to me! I rebased this branch. The changes related to the checkboxes are in [this commit](https://github.com/woocommerce/woocommerce-admin/pull/8514/commits/4cd824f89e03c8102d6ed78380c980a4cf7fbaee).

**Review by louwie17 — APPROVED** [review]
LGTM 🚀 Tested well, thanks for fixing this @octaedro 🎉

**Comment by octaedro** [comment]
Cherry-picked to 3.3.1

**Pull request closed without merging**